src/frontend/src/handler/create_index.rs

```diff
@@ -167,6 +169,12 @@ pub async fn handle_create_index(
         (graph, table)
     };
 
+    if table.owner != *session.user_name() {
+        let object = Object::TableId(table.id);
+        let action = Action::Create;
+        check_privilege(&session, &object, action)?;
```

Comment:
```suggestion
        check_privilege(&session, &Object::TableId(table.id), Action::Create)?;
```
Simplify into one liner?